### DISCUSSION TITLE
Migrating C# (VS) to Python for FLIR sensor

### DISCUSSION THREAD

**Discussion by spsaswat** [spsaswat/APPF_FLIR#8] [Polls]
Hi Team,
              I am considering migrating from C# to Python.
              
Below are some of the reasons:
- Python will allow cross-platform dev in Windows, Linux and Mac. 
-  Most of our team members are familiar with Python.
- The rest of the code uses Python

**Comment by wanqiqiu** [comment]
i think python would be better since we are all familiar with this programming language, it would be more effient and understandable for both developers and documentors.
